Find a functionally equivalent code:com.android.systemui.usb.StorageNotification.setMediaStorageNotification:COMMENT
Method Modifier: private     
Comment:/**
 * Sets the media storage notification.
 */

@@ -34,40 +34,43 @@
             pi = PendingIntent.getBroadcastAsUser(mContext, 0, intent, 0, UserHandle.CURRENT);
         }
         mMediaStorageNotification.icon = icon;
+        mMediaStorageNotification.color = mContext.getResources().getColor(com.android.internal.R.color.system_notification_accent_color);
         mMediaStorageNotification.setLatestEventInfo(mContext, title, message, pi);
+        mMediaStorageNotification.visibility = Notification.VISIBILITY_PUBLIC;
+        mMediaStorageNotification.category = Notification.CATEGORY_SYSTEM;
     }
     final int notificationId = mMediaStorageNotification.icon;
     if (visible) {

